docs/container.md

```diff
@@ -0,0 +1,69 @@
+---
+parent: Experimental features
+nav_order: 1
+---
+
+# ex-container
+
+For more information on this, see [CoreOS layering](https://github.com/coreos/enhancements/pull/7).
+
+rpm-ostree inherits work in [ostree-rs-ext](https://github.com/ostreedev/ostree-rs-ext/) to
+create "container native ostree" functionality.  This elevates OCI/docker containers to
+be natively supported as a transport mechanism for bootable operating systems.
+
+## Rebasing a client system
+
+Use this to switch to booting from a container image:
+
+```
+$ rpm-ostree rebase --experimental ostree-unverified-registry:quay.io/cgwalters/fcos
+```
+
+In the near future, the idea is to push an official container image as e.g.
+`quay.io/fedora/coreos:stable`.
+
+However, this model would just be using Docker/OCI transport "on the wire"
+for content that already exists today.  This would aid things like mirroring
+the OS alongside other container images, but for many users the next step
+is more interesting:
+
+## Using custom builds
+
+The ostree container functionality supports layered container images.
+See [fcos-derivation-example](https://github.com/cgwalters/fcos-derivation-example)
```

Comment:
I guess you can even embedded a sampler smaller version of that Dockerfile here, which should be enough to capture the juicy details. And then `rpm-ostree rebase` to it.

Reply:
Hmm.  Longer term I think this would probably show up in the Fedora (CoreOS) docs.  This is just kind of a temporary holding spot.  Further, we have some active debate about whether we promote a `Dockerfile` flow or [encourage this via butane](https://github.com/coreos/enhancements/pull/7/commits/f158aa5e9806a8e26c19d557d5bc9268e422e01a).

I think I'd rather keep fewer copies of this example flow for now, that OK?

Reply:
Sure. I was only reasoning about a random test repo more easily getting outdated / non-working.